lark/load_grammar.py

```diff
@@ -231,12 +231,11 @@ def will_not_get_removed(sym):
                 return not sym.name.startswith('_')
```

Comment:
We might want to think about changing this behavior. But I am not sure if there is a better solution...